README.md

```diff
@@ -53,9 +55,13 @@ repo-tools provides several options for generating an Appsody repository:
 ### 1) Composition of public stacks / repositories.
 If the stacks and repositories you are including are all publically available then repo-tools can simply compose a new repository file that uses references to the existing stack asset locations. When this type of build is required simply leave the `image-org` and `image-registry` fields of your configuration empty. The composed repository files will be stored in the `assets` folder generated when the tools are run.
 
-Further options to follow......
+### 2) Packaging private stacks / repositories.
+If your stacks / repositories are hosted in a private environment that your deployment environment and tools cannot access, such as GitHub Enterprise,  you can leverage the repo-tools to create an NGINX image that can serve the assets required to make use of your stacks from within the deployment environment. When this type of build is required configure the `image-org` field to be the name of the org within your target registry and the `image-registry` field to be the URL of the actual registry the images will be placed in. You can optionally configure the name of the resulting image using the `nginx-image-name` field. Once run your local docker registry will contain the generated NGINX image which can be pushed to the registry your deployment environment will access. Once deployed the image will server the repository index files that were created as part of the build.
 
-## Releasing the generated repositories.
-Once your index files are generated they simply need hosting in a location that can be accessed by your developer tools.
+## Running the tools
 
-To be continued.....
\ No newline at end of file
+To run the tools follow these staight forward steps:
+
+1) Create your configuration file and place it in the config folder.
+2) From the base folder of the repository run the build tool using the command `./scripts/hub_build.sh`.
```

Comment:
You need to specify the name of the config file as part of the command, ie ./scripts/hub_build.sh myconfig.yaml